pkg/solidity-utils/test/foundry/utils/BasePoolMathRoundingTest.sol

```diff
@@ -4,24 +4,27 @@ pragma solidity ^0.8.24;
 
```

Comment:
Interesting that you added this base test into utils folder, maybe I should do the same with other base tests.

Reply:
We have the same way in vault pkg